### ISSUE TITLE
[BUG] Error occurring when logging into game as a Monk.

### ISSUE THREAD

**Issue by Ookami313** [Hekili/hekili#2951]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### Describe the Issue

Running v10.2.0-1.0.2wtf-b. Logging into the game on my monk causes an error.

### How to Reproduce

Log into the game on my Monk.

### Player Information (Link)

https://pastebin.com/KJLrvAsp

### Error Messages (Link)

https://pastebin.com/pvQtygwu

**Comment by mbattersby** [comment]
```
Message: ...erface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua:2068: bad argument #1 to 'strformat' (number expected, got string)
Time: Fri Nov 17 17:58:37 2023
Count: 1
Stack: ...erface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua:2068: bad argument #1 to 'strformat' (number expected, got string)
[string "=[C]"]: in function `strformat'
[string "@Interface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua"]:2068: in main chunk

Locals: (*temporary) = "Use %s"
(*temporary) = "|W Flying Serpent Kick|w"
(*temporary) = "number expected, got string"
```

**Comment by BloodyFess** [comment]
> ```
> Message: ...erface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua:2068: bad argument #1 to 'strformat' (number expected, got string)
> Time: Fri Nov 17 17:58:37 2023
> Count: 1
> Stack: ...erface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua:2068: bad argument #1 to 'strformat' (number expected, got string)
> [string "=[C]"]: in function `strformat'
> [string "@Interface/AddOns/Hekili/Dragonflight/MonkWindwalker.lua"]:2068: in main chunk
> 
> Locals: (*temporary) = "Use %s"
> (*temporary) = "|W Flying Serpent Kick|w"
> (*temporary) = "number expected, got string"
> ```

The same error...

**Comment by Lionceaudor** [comment]
Hello,
Same here, thanks for your work !

https://pastebin.com/CEK2VgdT

**Comment by Hekili** [comment]
This should be resolved with v10.2.0-1.0.3.
